src/Common/MemoryTracker.cpp

```diff
@@ -175,7 +175,7 @@ void MemoryTracker::updatePeak(Int64 will_be)
 }
 
 
-void MemoryTracker::free(Int64 size)
+void MemoryTracker::free(Int64 size, VariableContext parent_level)
```

Comment:
You mean "initial level", not "parent level"?
Otherwise it will be confused with the `parent` field.

Reply:
Indeed (after submitting I already realized that the naming was bad, but did not want to trigger one more build in CI)